.gitlab-ci.yml

```diff
@@ -23,3 +24,54 @@ build-composer-dependencies:
     paths:
       - .composer
       - .Build
+
+php-lint-php7.4:
+  stage: lint
+  script:
+    - composer ci:php:lint
+  allow_failure: true
+
+php-lint-php7.3:
+  image: php:7.3
+  stage: lint
+  script:
+    - composer ci:php:lint
+  allow_failure: true
+
+php-lint-php7.2:
+  image: php:7.3
+  stage: lint
+  script:
+    - composer ci:php:lint
+  allow_failure: true
```

Comment:
I'd prefer to start with the lowest version first and then work our way up to 7.4.